Clean-up request IP validation (#21370)
directus/directus: .changeset/chatty-jobs-scream.md

---
"@directus/api": patch
---

> Cleaned-up IP validation on imports from URLs, requests in Flows and Webhooks to ensure the requests aren't incorrectly blocked

Author: ```suggestion
Cleaned-up and fixed IP validation used for external requests
```
